kong/init.lua

```diff
@@ -630,11 +630,13 @@ end
 
 
 function Kong.ssl_certificate()
+  local ctx = ngx.ctx
+
   kong_global.set_phase(kong, PHASES.certificate)
 
-  -- this doesn't really work across the phases currently (OpenResty 1.13.6.2),
-  -- but it returns a table (rewrite phase clears it)
-  local ctx = ngx.ctx
+  --TODO: doesn't work
```

Comment:
Is this a WIP or is something we will not ship in 2.4? Asking this as I see openresty changelog include some fixes for ngx.ctx in ssl_certificate phase.

Reply:
this pr has not cleaned up yet, yesterday we finally got needed stuff merged and test suite at least a bit more stable